source/kasko_js.rst

```diff
@@ -156,3 +156,56 @@ When testing is complete and you're ready to Go Live, please swap the
 Client TEST key for the Client LIVE key in your production site.
 
 .. note:: You must swap you client key with the LIVE client key before going live.
+
+Querystring Prefill
+-------------------
+
+Sometimes it's useful to prefill a webapp with predefined data. For example, an email campaign may have a link to the webapp integration. In order to store the email campaign tracker ID on the customer's policy, query string prefill can be used.
+
+.. note::   ?kdata=eyJmaXJzdF9uYW1lIjoiSm9obiJ9
+
+`kdata` is short for `KASKO data`. This querystring parameter is used to prefill an application with given `data` (name, address, email, etc) and `metadata` (could be anything, but most commonly used for analytics tracking data or agent information).
+
+.. warning::   `kdata` can only be used on the integration level. It will not work if set on webapp level (`webapp.kasko.io` domain). This is because KASKO JS is responsible for decoding `kdata` and passing it on to the webapp in a different format.
+
+`kdata` value can be a url-safe-base64-encoded string or a JSON string. **It is preferred to use url-safe-base64-encoded string as it is supported by all browsers.**
```

Comment:
Maybe it's valuable to show an example (maybe a link to online resource) of how to encode string to base64?

Reply:
Good point. Added that.